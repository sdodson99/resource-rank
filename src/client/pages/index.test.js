import 'regenerator-runtime';
import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import Index from './index';

describe('home page', () => {
  it('should render correctly', () => {
    const page = createRenderer().render(<Index />);

    expect(page).toMatchSnapshot();
  });
});
