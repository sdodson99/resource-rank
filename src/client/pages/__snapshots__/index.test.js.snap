// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page should render correctly 1`] = `
<Layout>
  <Hero />
  <FeatureListing
    features={
      Array [
        Object {
          "description": <div>
            <strong>
              Instantly find the best resources
            </strong>
             for learning about a specific topic.
          </div>,
          "imageSrc": "/img/clock.svg",
          "title": "Instant",
        },
        Object {
          "description": <div>
            <strong>
              Explore topics
            </strong>
             to find your next learning journey.
          </div>,
          "imageSrc": "/img/mag-glass.svg",
          "title": "Explore",
        },
        Object {
          "description": <div>
            <strong>
              Rate resources
            </strong>
             based on helpfulness for learning a topic.
          </div>,
          "imageSrc": "/img/star.svg",
          "title": "Rate",
        },
        Object {
          "description": <div>
            <strong>
              Add your own new topics and resources
            </strong>
             that are useful.
          </div>,
          "imageSrc": "/img/add.svg",
          "title": "Contribute",
        },
      ]
    }
  />
  <CallToAction />
</Layout>
`;
